addons/packages/ako-operator/1.4.0/README.md

```diff
@@ -36,9 +127,85 @@ The following configuration values can be set to customize the ako-operator inst
 | `avi_cni_plugin` | Optional | describes which cni plugin cluster is using. |
 | `avi_disable_static_route_sync` | Optional | describes ako should sync static routing or not. |
 | `avi_control_plane_ha_provider` | Required | describes whether Avi provides control plane HA service or not. |
-| `avi_management_cluster_vip_network_name` | Required | describes the data network name of the management cluster AKO will be deployed with. |
-| `avi_management_cluster_vip_network_cidr` | Required | describes the data network cidr of the management cluster AKO will be deployed with. |
+| `avi_management_cluster_vip_network_name` | Optional | describes the data network name of the management cluster AKO will be deployed with. |
+| `avi_management_cluster_vip_network_cidr` | Optional | describes the data network cidr of the management cluster AKO will be deployed with. |
 
 ## Usage Example
 
 This walkthrough guides you through using ako-operator...
+
+### Verify your configurations
+
+After the package is installed and reconcile successfully, two AKODeploymentConfigs are deployed globally.  
+You can list and inspect them with `kubectl get akodeploymentconfig`. They are the configuration interfaces for the ako-operator.
+In general,
+
+* `install-ako-for-management-cluster` specifies the parameters to deploy AKO statefulset in the management cluster, which
+  then provides L4/L7 load balancing for applications.
+* `install-ako-for-all` specifies the parameters to deploy AKO statefulsets and reconcile users in each future workload clusters. The ako-operator
+  passes these parameters to each workload clusters via addon secrets.
+
+In the management cluster, check if AKO statefulset has been deployed successfully and also check AKO's log to make sure no persistent error messages.
+
+```shell
+# retrieve the pod name of the ako-operator manager
+kubectl get pods -n tkg-system-networking
+
+# check if the log is error-free.
+kubectl logs ako-operator-controller-manager-xxxxxxxx-xxxxx manager -n tkg-system-networking
+```
+
```

Comment:
There should be one section to tell how to leverage `akodeploymentconfig`'s `spec.clusterSelector` to per-cluster configure and deploy AKO to enable AKO in different modes or functionality.